extensions/amp-action-macro/0.1/amp-action-macro.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {LayoutPriority} from '../../../src/layout';
+
+/**
+* The <amp-action-macro> element is used to define a reusable action macro.
+*/
+export class AmpActionMacro extends AMP.BaseElement {
```

Comment:
you probably wanna add the buildCallback and assert the `amp-action-micro` experiment